.github/workflows/auto-build-rpm.yml

```diff
@@ -50,7 +60,7 @@ jobs:
           if [ "$VERSION" != "merge" && "$VERSION" != "master" ];then
             export checkout=release/${VERSION}
           fi
-          make package type=rpm app=dashboard version=${VERSION} checkout=${checkout} image_base=centos image_tag=7 local_code_path=./apisix-dashboard
+          make package type=rpm app=dashboard version=${VERSION} checkout=${checkout} image_base=centos image_tag=7 local_code_path=./apisix-dashboard buildx=True
```

Comment:
Hi, please bump the apisix-build-tools to version v2.2.0, which supports the new argument `buildx=True`. Thanks.